pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -22,6 +22,9 @@ type Configuration struct {
 
 	// Storage is the optional storage configuration
 	Storage Storage `json:"storage,omitempty" yaml:"storage,omitempty"`
+
+	// ServiceAccount is the service account under which scorecard tests are run. This field is optional. If left unset, the `default` service account will be used.
```

Comment:
nit: I really dislike LONG comments. I usually wrap comments to make them easier to read. I will not block this PR for that just more for edification :)